Bottom: 58c22893007093a416fe472a2c7fc2e0b64de36e
Top:    f04f1ae7336755f20bd81b5a8c57c235e80a911b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 15:38:43 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 07b1d99..89ee1e8 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -65,20 +65,20 @@ ghid_set_layer (const char *name, int group, int empty)
 {
   render_priv *priv = gport->render_priv;
   int idx = group;
-  if (idx >= 0 && idx < max_layer)
+  if (idx >= 0 && idx < max_group)
     {
       int n = PCB->LayerGroups.Number[group];
       for (idx = 0; idx < n-1; idx ++)
 	{
 	  int ni = PCB->LayerGroups.Entries[group][idx];
-	  if (ni >= 0 && ni < max_layer + 2
+	  if (ni >= 0 && ni < max_copper_layer + 2
 	      && PCB->Data->Layer[ni].On)
 	    break;
 	}
       idx = PCB->LayerGroups.Entries[group][idx];
     }
 
-  if (idx >= 0 && idx < max_layer + 2)
+  if (idx >= 0 && idx < max_copper_layer + 2)
     {
       priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
